api/metadata/permissions.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.permissions import IsAuthenticated
+
+
+class MetadataFieldPermissions(IsAuthenticated):
+    def has_permission(self, request, view):
+        if not super().has_permission(request, view):
+            return False
+
+        organisation = request.data.get("organisation")
+        if view.action == "create" and request.user.belongs_to(organisation):
+            return request.user.is_organisation_admin(request.data.get("organisation"))
```

Comment:
I think you can replace `request.data.get("organisation")` with just `organisation` since you've already stored it in a variable above. 

Also, it might be worth creating an organisation level permission for this (although perhaps that's something that we can do in a separate PR).